doc/luasnip.txt

```diff
@@ -596,29 +553,27 @@ ChoiceNodes allow choosing between multiple nodes.
 <
 
 
-`c()` expects as its first arg, as with any jumpable node, its position in the
-jumplist, and as its second a table with nodes, the choices. This table can
-either contain a single node or a table of nodes. In the latter case the table
-will be converted into a `snippetNode`. The third parameter is a table of
-options with the following keys:
-
-
-- `restore_cursor`: `false` by default. If it is set, and the node that was
-    being edited also appears in the switched-to choice (can be the case if a
-    `restoreNode` is present in both choice) the cursor is restored relative to
-    that node.
-    The default is `false` as enabling might lead to worse performance. It’s
-    possible to override the default by wrapping the `choiceNode`-constructor
-    in another function that sets `opts.restore_cursor` to `true` and then using
-    that to construct `choiceNode`s:
-    `lua     local function restore_cursor_choice(pos, choices, opts)         if opts then             opts.restore_cursor = true         else             opts = {restore_cursor = true}         end         return c(pos, choices, opts)     end`
-
+`c(jump_index, choices, node_opts)` - |luasnip-`jump_index`|: `number`, since
```

Comment:
here (I think searching for ` - ` (a dash surrounded by spaces) can find you all inlined lists)